src/app/screens/LNURLChannel/index.tsx

```diff
@@ -73,8 +76,12 @@ function LNURLChannel() {
         msg.reply(callbackResponse?.data);
       }
     } catch (e) {
-      console.error(e);
-      if (e instanceof Error) toast.error(`Error: ${e.message}`);
+      if (axios.isAxiosError(e)) {
```

Comment:
Why only if it's an axios error? if e.g. api.getInfo() fails there is no error shown?